### ISSUE TITLE
Create torrents for bulk data

### ISSUE THREAD

**Issue by ndawg** [unitedstates/congress#226]
Right now, I'm using the `fdsys` script to scrape all bill texts for every Congress session that has data. This takes a long, long time, so having the data hosted somewhere makes sense. After all, bills from previous congressional sessions aren't going to be modified. However, it is about a gigabyte of data per session, so no host would make sense - on the other hand, this is a great use case for torrents. The main issue is that you would most likely end up being stuck with all the formats possible in one torrent, but that's okay for me. Thoughts on this?

**Comment by konklone** [comment]
Sunlight used to host these on S3, but doesn't do that anymore.

It is a pretty decent use case for torrents, though I don't know if any of the organizers here have (or are familiar with) torrent management software, or want to take on the maintenance.

**Comment by dwillis** [comment]
@konklone, do you happen to know what hosting these on S3 cost Sunlight?

**Comment by konklone** [comment]
No, I don't remember anymore...not even the order of magnitude. If it was hugely expensive I'd probably remember, but we also didn't promote them very well -- they are just linked to on the wiki.

And actually, they still are:
https://github.com/unitedstates/congress/wiki

And the Sunlight downloads...still work. They're just not updated anymore. And are delivered over plain HTTP (gross).
